tests/TestFiles/Show.hs

```diff
@@ -1,5 +1,8 @@
 module Show where
 
+show0 :: String
+show0 = show (0 :: Int)
```

Comment:
Was this useful in debugging this branch?  If so let's add it to the tests:
https://github.com/BillHallahan/G2/blob/master/tests/Test.hs#L404-L409